tests/serverpod_test_server/test/serialization_test.dart

```diff
@@ -17,6 +20,30 @@ void main() {
     expect(unpacked.aStringifiedEnum, isNull);
   });
 
+  test(
+      'Given a server-side enum serialized as string value when serializing then the value is unpacked correctly',
+      () {
+    var types = server.Types(aStringifiedEnum: server.TestEnumStringified.one);
+
+    var encoded = SerializationManager.encode(types);
```

Comment:
Question: Is the SerializationManager shared between the client and the server? Otherwise, should you use a server side only serialization manager to make this test consistent?

Reply:
it is.